### ISSUE TITLE
can not conncet to the metamask,and walletconnect does not work---desktop latest

### ISSUE THREAD

**Issue by leverwwz** [ethereum/remix-desktop#77]
my os is win10, 

can not conncet to the metamask,and walletconnect does not work---desktop latest version

**Comment by mLjH** [comment]
have this issue too. nothing happens when i scan the walletconnect QR code with metamask mobile on my iPhone.

**Comment by yann300** [comment]
can you try again the wallet connect plugin, we've updated it.

**Comment by frankzielen** [comment]
Even with my infura ID (free account) it doesn't work. Both iOS wallets, MetaMask and Coinbase, do not accept the QR code (MetaMask: nothing happens after scanning; Coinbase: message "Invalid QR code"). I have selected Ethereum mainnet and Kovan testnet in apps before scanning - same behaviour. Any ideas? And finally, even if it would work with infura ID - would I be able to connect my wallet for BSC Smart Chain?

**Comment by Nasfame** [comment]
Oh My God!!! no real progress on this. Can't we integrate an inbuilt wallet similar to metamask?

**Comment by yann300** [comment]
issue is still opened but I believe the wallet connect plugin has been fixed by now, could you give it a try?

**Comment by Nasfame** [comment]
Yes, wallet connect plugin works. I was able to integrate metamask by scanning the qr code on my smartphone.

But its a bit inconvenient compared and not reliable
